pkg/provider/resource_cluster.go

```diff
@@ -88,77 +88,77 @@ func (c *ClusterResource) Schema() *schema.Resource {
 			},
 			"created_at": {
-				Description: "Cluster Creation Time",
+				Description: "Cluster Creation Time.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
 			"deleted_at": {
-				Description: "Cluster Deletion Time",
+				Description: "Cluster Deletion Time.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
 			"expired_at": {
-				Description: "Cluster Expiry Time",
+				Description: "Cluster Expiry Time.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
 			"first_recoverability_point_at": {
-				Description: "Earliest Backup recover time",
+				Description: "Earliest Backup Recover Time.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
 			"instance_type": {
-				Description: "Instance Type",
+				Description: "Instance Type. For example, \"azure:Standard_D2s_v3\", \"aws:c5.large\".",
 				Type:        schema.TypeString,
 				Required:    true,
 			},
 			"cluster_id": {
-				Description: "cluster ID",
+				Description: "Cluster ID.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
 			"connection_uri": {
-				Description: "cluster connection uri",
+				Description: "Cluster connection URI.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
 			"ro_connection_uri": {
-				Description: "cluster read only connection uri",
+				Description: "Cluster read-only connection URI. Only available for high availability clusters.",
 				Type:        schema.TypeString,
 				Computed:    true,
 			},
 			"password": {
-				Description: "Password",
+				Description: "Password for the user edb_admin. It must be 12 characters or more.",
 				Type:        schema.TypeString,
 				Required:    true,
 				Sensitive:   true,
 			},
 			"pg_config": {
-				Description: "Instance Type",
+				Description: "Database Configuration Parameters. See https://www.enterprisedb.com/docs/biganimal/latest/using_cluster/03_modifying_your_cluster/05_db_configuration_parameters/ for details.",
 				Type:        schema.TypeList,
 				Optional:    true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"name": {
-							Description: "GUC Name",
+							Description: "GUC Name.",
 							Type:        schema.TypeString,
 							Required:    true,
 						},
 						"value": {
-							Description: "GUC Value",
+							Description: "GUC Value.",
 							Type:        schema.TypeString,
 							Required:    true,
 						},
 					},
 				},
 			},
 			"pg_type": {
-				Description: "Postgres type",
+				Description: "Postgres type. For example, \"epas\", \"pgextended\" or \"postgres\".",
 				Type:        schema.TypeString,
 				Required:    true,
 			},
 			"pg_version": {
-				Description: "Postgres Version",
+				Description: "Postgres version. See https://www.enterprisedb.com/docs/biganimal/latest/overview/05_database_version_policy/#supported-postgres-types-and-versions for supported Postgres types and versions.",
```

Comment:
So, this description here in the `pkg/provider/resource_cluster.go` generates the `docs/resources/cluster.md` documentation. 
Above, [in this suggestion](https://github.com/EnterpriseDB/terraform-provider-biganimal/pull/74#discussion_r1033603509), the link text is `[Database version policy]` instead of `[Supported Postgres types and versions]`. Which one should I select, @dwicinas?